lib/ably/AblyRest.php

```diff
@@ -124,7 +125,8 @@ public function authorise( $options = array(), $params = array(), $force = false
             if ( $this->token->expires > $this->timestamp() ) {
                 if ( !$force ) {
                     # using cached token
-                    $this->log_action( 'authorise()', sprintf("\tusing cached token; expires = %s\n\tfor humans token expires on %s", $this->token->expires, gmdate("r",$this->token->expires)) );
+                    $this->log_action('authorise()', sprintf("\tusing cached token; expires = %s\n\tfor humans token expires on %s",
```

Comment:
Not my code, so I don't know what were the intentions. PHP doesn't have any built in logging system, it's up to the developer.
It could be done much more cleanly, for instance you could infer the method name from callstack and the logger should be a separate (singleton) class.